src/Shikimori/Provider.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace SocialiteProviders\Shikimori;
+
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'SHIKIMORI';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = [
+        'user_rates'
+    ];
+
+    protected $scopeSeparator = ' ';
+
+    protected function getShikimoriUrl()
+    {
+        return $this->getConfig('base_url');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function additionalConfigKeys()
+    {
+        return ['base_url'];
```

Comment:
Are you really sure about this ?
By reading your docs it seems that `base_url` will always be `https://shikimori.one`

Reply:
@M9snikFeed